### ISSUE TITLE
Allow custom input

### ISSUE THREAD

**Issue by vinerz** [ellipticaljs/paper-autocomplete#62]
This element is awesome, congratulations for its contributors!

However, my usage requires that the element to not be constrained by the source values, allowing custom inputs there too.

This way, the autocomplete should act only as a helper to get to the result faster.

Can it be done today with some config property? I am currently using `autocompleteElement.$.autocompleteInput.value` to accept any value, but this is a really dirty workaround.

**Comment by jhuesos** [comment]
I agree I think it would be useful. How do you expect this should work? What would you expect to happen with the value? value === text === value in the input field?
